docs/source/developer/installer/INSTALLER.md

```diff
@@ -0,0 +1,80 @@
+# Creating an Installer
+
+During generation, CMake should produce an `install` target and a `package` target.
+
+The `install` target will copy the High Fidelity targets and their dependencies to your `CMAKE_INSTALL_PREFIX`.  
+This variable is set by the `project(hifi)` command in `CMakeLists.txt` to `C:/Program Files/hifi` and stored in `build/CMakeCache.txt`
```

Comment:
Lots of "HiFi" stuff in here - out of date?

Reply:
This needs to be updated in the main repo. The file will automatically be pulled from there. https://github.com/kasenvr/project-athena/pull/242